modules/fuse/fuse_impl.h

```diff
@@ -1,3 +1,4 @@
+// accredit to the hdfs fuse file strcuture design, nice design
```

Comment:
I cannot see how this file `fuse_impl.h` related to hdfs ones.

Reply:
oh that's true, I forget to delete it